BuildScript.Nuke/Components/Tasks/CompileTask.cs

```diff
@@ -145,6 +145,7 @@ private static string GetToolPath (string msBuildNukePath, string msBuildPath, V
           .SetProperty(MSBuildProperties.CompanyUrl, assemblyMetadata.CompanyUrl)
           .SetProperty(MSBuildProperties.ProductName, assemblyMetadata.ProductName)
           .SetProperty(MSBuildProperties.AssemblyOriginatorKeyFile, directories.SolutionKeyFile)
+          .SetProperty(MSBuildProperties.ContinuousIntegrationBuild, "true")
```

Comment:
Am not happy with the name. CI build is something else than a full build, from the semantics / intention. Technically, we have implemented them both via build script. Do you have additional naming suggestions?

Reply:
That is a dotnet SDK name :)